lesson02/assignment/index.html

```diff
@@ -3,12 +3,36 @@
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
+        <link rel="stylesheet" href="css/main.css" />
         <title>Assignment 02</title>
     </head>
     <body>
         <main class="main">
-            <h1>title</h1>
+
+         <h1>hello</h1>
+              <div class="main-div">
+               <div class="card">
+                 <div class="left-content-card">
+                   <img src="img/unsplash-headshot.jpg" alt="male avatar picture for employee id">
```

Comment:
Is the new lesson02/assignment/img/avatar.jpg image being used anywhere? If not, let's delete it.